src/components/Input/Input.module.css

```diff
@@ -16,11 +16,18 @@
 }
 
 .input.isInvalid,
-.input:invalid {
+.input:user-invalid {
   background: var(--color-background-accent-darken);
   border-bottom-color: var(--color-alert-darken);
 }
 
+@supports not (selector(:user-invalid)) {
```

Comment:
[memo]
Configuration for Safari 16.4 and below that does not yet support user-invalid.
Note that @supports: selector() has been supported since Safari 14.1 and all other browsers.

user-invalidに未対応のSafari 16.4未満用の設定。
なお、 @supports: selector() は Safari 14.1から対応していて、他の全ブラウザにも対応済み。